File: IronVault/UI/ironvault_mobile/lib/models/clanarina.dart
import 'package:json_annotation/json_annotation.dart';

part 'clanarina.g.dart';

@JsonSerializable()
class Clanarina {
  int? clanarinaId;
  String? naziv;
  double? cijena;
  String? opis;

  Clanarina({this.clanarinaId, this.naziv});

  factory Clanarina.fromJson(Map<String, dynamic> json) =>
      _$ClanarinaFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$ClanarinaToJson(this);
}

File: IronVault/IronVault.Model/Models/Clanarina.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class Clanarina
    {
        public int ClanarinaId { get; set; }

        public string Naziv { get; set; } = null!;

        public float Cijena { get; set; }

        public string Opis { get; set; } = null!;
    }
}
